toe/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+test.hoon
```

Comment:
Added `node_modules` and `dist` to [master](https://github.com/urbit/examples/blob/master/.gitignore). So it should be fine to remove this file?

Reply:
whoops... yeah, that shouldn't be there (I had already deleted that test.hoon but forgot about this)